test/parallel/test-fs-promises-readfile-empty.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+require('../common');
+
+const assert = require('assert');
+const { promises: fs } = require('fs');
+const fixtures = require('../common/fixtures');
+
+const fn = fixtures.path('empty.txt');
```

Comment:
Can you add `common.crashOnUnhandledRejection()` here? Or wrap the promise callbacks in `common.mustCall()`, otherwise if the promise rejects the tests won't fail.